CONTRIBUTING.md

```diff
@@ -36,9 +36,9 @@ git clone https://github.com/<your_github_username>/chakra-ui.git
 cd chakra-ui
 ```
 
-3. Setup all dependencies and build. Chakra UI uses `yarn`, so simply run
-   `yarn bootstrap`. This command will install dependencies, bootstrap the repo
-   using `lerna` and build all packages.
+3. Setup all dependencies and build. Chakra UI uses `yarn` and `lerna`, so
+   simply run `yarn bootstrap`. This command will install dependencies,
+   bootstrap the repo using `lerna` and build all packages.
```

Comment:
```suggestion
3. Setup all dependencies and build. Chakra UI uses `yarn` and `lerna`, so run
   `yarn bootstrap`. This command will install dependencies, bootstrap the repo
   using `lerna` and build all packages.
```

I know this wasn't added  by you, but we might as well get rid of it while we're here.

[Don't Say Simply](https://www.youtube.com/watch?v=gsT2BBWBVmM)